google/cloud/sqlalchemy_spanner/sqlalchemy_spanner.py

```diff
@@ -499,14 +506,20 @@ def get_columns(self, connection, table_name, schema=None, **kw):
             columns = snap.execute_sql(sql)
 
             for col in columns:
-                cols_desc.append(
-                    {
-                        "name": col[0],
-                        "type": self._designate_type(col[1]),
-                        "nullable": col[2] == "YES",
-                        "default": None,
+                col_desc = {
+                    "name": col[0],
+                    "type": self._designate_type(col[1]),
+                    "nullable": col[2] == "YES",
+                    "default": None,
+                }
+
+                if col[3] is not None:
+                    col_desc["computed"] = {
+                        "persisted": True,
+                        "sqltext": col[3],
                     }
-                )
+                cols_desc.append(col_desc)
+
```

Comment:
Some changes of the reflection method are required, indeed like this: if the column is not generated, `computed` field shoud not exist in the result (checked by the tests).